js/data/cryptoListingCurrencies.js

```diff
@@ -28,10 +28,10 @@ const baseCurrencies = [
   'ETH',
   'FCT',
   'HSR',
-  'IOTA',
   'KMD',
   'LSK',
   'LTC',
+  'MIOTA',
   'MONA',
```

Comment:
In the app, you'll still see both MIOTA and IOTA because one comes from btc avg and the other from CMC and they both refer to the same coin. @tyler-smith will update the ticker to only use CMC's MIOTA.